### ISSUE TITLE
Repeated tip

### ISSUE THREAD

**Issue by innerlee** [JuliaEditorSupport/julia-vscode#199]
![screenshot_2017-07-02_21-48-04](https://user-images.githubusercontent.com/9464825/27770473-e8ba0b06-5f71-11e7-877e-73e8667c8800.png)

There are 9 numbers in the tip. I guess it may relate to the server restarts, as in the following output:
```bash
[Info  - 5:30:18 PM] Connection to server got closed. Server will restart.
```
However, there are 15 restart messages, but only 9 repeats.

---
edit:
There are indeed 16 repeats. Missed the scroll bar earlier.
![screenshot_2017-07-02_22-06-54](https://user-images.githubusercontent.com/9464825/27770536-e42add62-5f72-11e7-96a7-e10365f2fca9.png)

**Comment by ZacLN** [comment]
Yep your analysis is correct, I'll dig into what's causing it

**Comment by davidanthoff** [comment]
Is this still a problem with the latest version of VS Code?

**Comment by ZacLN** [comment]
Yep, I forgot to actually investigate this but I'll open an issue

**Comment by ZacLN** [comment]
https://github.com/Microsoft/vscode/issues/31915

**Comment by ZacLN** [comment]
upstream issue was moved to [here](https://github.com/Microsoft/vscode-languageserver-node/issues/246#issuecomment-326591800), solved by updating vscode-languageclient to v3.3.0
